dev-lang/rust/rust-1.29.0.recipe

```diff
@@ -0,0 +1,169 @@
+SUMMARY="Modern and safe systems programming language"
+DESCRIPTION="Rust is a systems programming language that runs blazingly fast, \
+prevents almost all crashes*, and eliminates data races."
+HOMEPAGE="https://www.rust-lang.org/"
+COPYRIGHT="2018 The Rust Project Developers"
+LICENSE="MIT"
+REVISION="1"
+cargoVersion="0.30.0"
+rlsVersion="0.130.0"
+rustfmtVersion="0.99.1"
+clippyVersion="0.0.212"
+SOURCE_URI="https://static.rust-lang.org/dist/rustc-$portVersion-src.tar.xz"
+CHECKSUM_SHA256="3943da98fb478a336ede7404e42ff76ef6ba4fc2b82012cfccd6b9fc4bd2c191"
+SOURCE_DIR="rustc-$portVersion-src"
+PATCHES="rust-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	rust_bin$secondaryArchSuffix = $portVersion
```

Comment:
I guess you meant:
```diff
-	rust_bin$secondaryArchSuffix = $portVersion
+	rust$secondaryArchSuffix = $portVersion
```
(Everything else looks OK but I did not try the recipes yet.)

Reply:
Good catch, this is a copy and paste error. Will fix.